indico/modules/users/models/users.py

```diff
@@ -624,6 +626,8 @@ def make_email_primary(self, email):
         db.session.flush()
         secondary.is_primary = True
         db.session.flush()
+        kwargs = dict(old=self._primary_email.email, new=secondary.email)
+        signals.users.primary_email_changed.send(self, **kwargs)
```

Comment:
```suggestion
        signals.users.primary_email_changed.send(self, old=self._primary_email.email, new=secondary.email)
```

Reply:
Also getting the old value here seems to be a bit fragile. Why not set `old = self.email` before changing it and use `email` for new?